logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultChronoZonedDateTimeAssertions.kt

```diff
@@ -44,4 +47,86 @@ class DefaultChronoZonedDateTimeAssertions : ChronoZonedDateTimeAssertions {
     ): Assertion = container.createDescriptiveAssertion(IS_EQUAL_TO, expected) {
         it.isEqual(expected)
     }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isBefore(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isBefore(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isBeforeOrEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isBeforeOrEqual(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isAfter(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isAfter(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isAfterOrEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isAfterOrEqual(parseZonedDateTime(expected))
+    }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isEqual(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isEqual(parseZonedDateTime(expected))
+    }
+
+    private fun parseZonedDateTime(data: String): ZonedDateTime {
+        /**
+         * The specification of the zone can look like this [Europe/Paris]
+         * and needs to be handled by the native jdk implementation.
+         */
+        if (data.indexOf('[') > -1 && (data.indexOf('[') < data.indexOf(']'))) {
+            return ZonedDateTime.parse(data)
+        }
+
+        val formatter = DateTimeFormatterBuilder()
+            .parseCaseSensitive()
+            .append(DateTimeFormatter.ISO_LOCAL_DATE)
+            .optionalStart().appendLiteral('T').append(DateTimeFormatter.ISO_LOCAL_TIME).optionalEnd()
+            .optionalStart().appendLiteral('Z').optionalStart().appendOffsetId().optionalEnd().optionalEnd()
```

Comment:
Sorry, totally my fault, I made a mistake in the description. We want to accept either
` Z`  as time zone or `+hh:mm`  or `-hh:mm` and not `Z+hh:mm` or `Z-hh:mm`

Reply:
Ah ok. Yea sure, I'll adjust it